Lib/test/datetimetester.py

```diff
@@ -279,7 +280,7 @@ def test_constructor(self):
             tz = timezone(subminute)
             self.assertNotEqual(tz.utcoffset(None) % timedelta(minutes=1), 0)
         # invalid offsets
-        for invalid in [timedelta(1, 1), timedelta(1)]:
+        for invalid in [timedelta(1, 1), timedelta(days=1, microseconds=1)]:
```

Comment:
Why does this need to be changed?

Reply:
Sorry, looks like i forgot to restore it